client/src/index.css

```diff
@@ -62,10 +62,25 @@ body, html, .App, #root, .outer {
   text-align: right;
   font-size: 13px;
   padding-top: 10px;
-  color: #7f7d7d;
+  /*color: #7f7d7d;*/
```

Comment:
both of these colors were commented out in that merge. One should be deleted and one should be used